Form/form_gb.xml

```diff
@@ -167,7 +167,7 @@
             </column>
         </section>
     </form>
-    <form id='UK1841' type='Census' title='1841 England and Wales Census' date='6 Jun 1841'>
+    <form id='UK1841' type='Census' title='1841 England and Wales Census' date='6 Jun 1841' reference='HO 107//, folio , page '>
```

Comment:
The reference should conform to the WHERE_WITHIN_PAGE format on page 64 of the GEDCOM 5.5.1 specification.

Use:  'Piece:HO 107//, Folio: , Page: '

Reply:
I based these on The National Archives [guidance](https://www.nationalarchives.gov.uk/help-with-your-research/research-guides/census-records/). They give an example on that page

> The National Archives reference for this page, from the 1891 census, is RG 12/1822, Folio 156, Page 5.

I can follow GEDCOM rules but isn't there a danger that we present text structued to be machine readable in a field that is meant to be human readable? Even given that statement, I can see benefits of the GEDCOM approach.

If using GEDCOM should we separate out Department, Series and Piece?
Maybe include Schedule too?

```suggestion
1841
Department: HO, Series: 107, Piece: , Book: , Folio: 

1851
Department: HO, Series: 107, Piece: , Folio: , Page: , Schedule: 

1861
Department: RG, Series: 9, Piece: , Folio: , Page: , Schedule: 
```

For future reference, I believe @Nick-Hall meant WHERE_WITHIN_SOURCE in the GEDCOM 5.5.1 specification

Reply:
References now conform to WHERE_WITHIN_SOURCE format from GEDCOM